Allows to read and write "Player properties" from LocalStorage & Adds a camera API (#1525)
thecodingmachine/workadventure: docs/maps/api-room.md

    name: string;       // A unique name for this iframe
    url: string;        // The URL the iframe points to.
    position: {

>         x: number,      // relative to the map or player coordinates, depending on origin
>         y: number,      // relative to the map or player coordinates, depending on origin
>         width: number,  // In pixels, sensitive to zoom level and scale
>         height: number, // In pixels, sensitive to zoom level and scale

Reviewer: ```suggestion
        x: number,      // In "game" pixels, relative to the map or player coordinates, depending on origin
        y: number,      // In "game" pixels, relative to the map or player coordinates, depending on origin
        width: number,  // In "game" pixels
        height: number, // In "game" pixels
```

    },
    visible?: boolean,  // Whether to display the iframe or not
    allowApi?: boolean, // Whether the scripting API should be available to the iframe
    allow?: string,     // The list of feature policies allowed

>     origin: "player" | "map" // The origin used to place the x and y coordinates of the iframe's top-left corner
>     scale: number, // A ratio used to resize the iframe (will affect the iframe's width and height when it is rendered

Reviewer: ```suggestion
    origin: "player" | "map" // The origin used to place the x and y coordinates of the iframe's top-left corner, defaults to "map"
    scale: number, // A ratio used to resize the iframe
```

Use `WA.room.website.delete` to completely remove an embedded website from your map.

### The EmbeddedWebsite class

Instances of the `EmbeddedWebsite` class represent the website displayed on the map.

```typescript
class EmbeddedWebsite {
  readonly name: string;
  url: string;
  visible: boolean;
  allow: string;
  allowApi: boolean;

>   x: number;         // In pixels, relative to the map or player coordinates, depending on origin
>   y: number;         // In pixels, relative to the map or player coordinates, depending on origin
>   width: number;     // In pixels, sensitive to zoom level and scale
>   height: number;    // In pixels, sensitive to zoom level and scale

Reviewer: ```suggestion
  x: number;         // In "game" pixels, relative to the map or player coordinates, depending on origin
  y: number;         // In "game" pixels, relative to the map or player coordinates, depending on origin
  width: number;     // In "game" pixels
  height: number;    // In "game" pixels
```
